core/lib/server/pub.ts

```diff
@@ -999,7 +891,13 @@ export const updatePub = async ({
 	const result = await maybeWithTrx(db, async (trx) => {
 		// Update the stage if a target stage was provided.
 		if (stageId !== undefined) {
-			await movePub(pubId, stageId, trx).execute();
+			try {
+				await movePub(pubId, stageId, trx).execute();
+			} catch (err) {
+				if (!isUniqueConstraintError(err)) {
+					throw err;
+				}
+			}
```

Comment:
what is this supposed to catch? you moving it to the stage it's already in?

Reply:
yeah exactly. previously that was taken care of (incorrectly) by an `on conflict do nothing` clause. now we just catch those errors and continue since there's no need to halt just because the stage is unchanged